packages/sycamore-router/Cargo.toml

```diff
@@ -27,5 +27,6 @@ features = [
   "Location",
   "PopStateEvent",
   "Window",
+  "KeyboardEvent",
```

Comment:
Nit: could you please specify the features in alphabetical order

Reply:
done!